DUI3-DX/Connectors/Rhino/Speckle.Connectors.Rhino7/Operations/Send/SendOperation.cs

```diff
@@ -0,0 +1,52 @@
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Speckle.Connectors.DUI.Bindings;
+using Speckle.Core.Models;
+
+namespace Speckle.Connectors.Rhino7.Operations.Send;
+
+/// <summary>
+/// Stateless send operation orchestrator.
+/// </summary>
+public sealed class SendOperation
+{
+  private readonly RootBaseObjectBuilder _baseBuilder;
+  private readonly IBaseObjectSender _baseObjectSender;
+
+  public SendOperation(RootBaseObjectBuilder baseBuilder, IBaseObjectSender baseObjectSender)
+  {
+    _baseBuilder = baseBuilder;
+    _baseObjectSender = baseObjectSender;
+  }
+
+  /// <summary>
+  /// Executes a send operation given information about the host objects and the destination account.
+  /// </summary>
+  /// <param name="sendFilter"></param>
+  /// <param name="accountId"></param>
+  /// <param name="projectId"></param>
+  /// <param name="modelId"></param>
+  /// <param name="onOperationProgressed"></param>
+  /// <param name="ct"></param>
+  /// <returns></returns>
+  public async Task<string> Execute(
+    ISendFilter sendFilter,
+    string accountId,
+    string projectId,
+    string modelId,
+    Action<string, double?>? onOperationProgressed = null,
+    CancellationToken ct = default
+  )
+  {
+    Base commitObject = _baseBuilder.Build(sendFilter, onOperationProgressed, ct);
+
+    // base object handler is separated so we can do some testing on non-production databases
+    // exact interface may want to be tweaked when we implement this
+    string versionId = await _baseObjectSender
```

Comment:
It would be nice to keep in mind we will not return only `versionId` from seriliazer after improvements on caching `ObjectReference`s. It means that we need to think where we will store these objects as state